hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2198,7 +2198,11 @@
       name: chip_sw_entropy_src_cs_aes_halt
       desc: '''Verify the aes halt handshake with CSRNG.
 
-            Details TBD.
+            - Check that the AES block halts when the edn signals that sufficient entropy isn't
+              available to reseed via assertions.
+            - TODO: If the above assertion isn't covered... modify the above test to consume
+              entropy more quickly, by writing more quickly to the AES and KMAC and possibly
+              triggering reseeds by touching a block repeatedly.
```

Comment:
it may be worth clarifying that this test is talking about the AES engine within CSRNG, not the AES module itself. If you are ok with it, I will reword it a bit more.

Reply:
Oh, yeah, I had this very much wrong then, I assumed (and never saw anything that broke my assumption) that this test was that the AES block would halt if entropy wasn't available for masking. If you can correct that in the testplan, I'll do my best to update the issue that had me confused.

Reply:
Oh, I wasn't aware that this is about the AES engine inside CSRNG. Thanks for catching this @sriyerg .